### ISSUE TITLE
Undefined method 'repo'

### ISSUE THREAD

**Issue by abinoam** [mmozuras/pronto-poper#5]
Trying to use pronto-popper and got this error.

```
/home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/pronto-0.6.0/lib/pronto/runner.rb:26:in `repo_path': undefined method `repo' for nil:NilClass (NoMethodError)
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/pronto-poper-0.6.0/lib/pronto/poper.rb:9:in `run'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/pronto-0.6.0/lib/pronto/runners.rb:16:in `block in run'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/pronto-0.6.0/lib/pronto/runners.rb:13:in `each'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/pronto-0.6.0/lib/pronto/runners.rb:13:in `run'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/pronto-0.6.0/lib/pronto.rb:42:in `run'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/pronto-0.6.0/lib/pronto/cli.rb:52:in `run'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/gems/pronto-0.6.0/bin/pronto:6:in `<top (required)>'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/bin/pronto:23:in `load'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/bin/pronto:23:in `<main>'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/bin/ruby_executable_hooks:15:in `eval'
	from /home/abinoam/.rvm/gems/ruby-2.3.0@gomedical/bin/ruby_executable_hooks:15:in `<main>'
```

Gemfile.lock

```ruby
    pronto (0.6.0)
      gitlab (~> 3.6, >= 3.4.0)
      octokit (~> 4.3, >= 4.1.0)
      rugged (~> 0.24, >= 0.23.0)
      thor (~> 0.19.0)
    pronto-poper (0.6.0)
      poper (~> 0.1.0)
      pronto (~> 0.6.0)
```

**Comment by abinoam** [comment]
Trying to fix it

**Comment by abinoam** [comment]
The bug is in popper. Fixing it.

**Comment by abinoam** [comment]
Patched on pronto repo PR mmozuras/pronto#153
